src/openagi/memory/STMemory.py

```diff
@@ -0,0 +1,43 @@
+from .base import BaseMemory
+import uuid
+from typing import Any, Dict, List
+
+class STMemory(BaseMemory):
```

Comment:
Refactor LT & STMemory as per the discussion. They should not be responsible for storing certain type of data only. It should accept generic data.